source/systems/cloud_user_guide.rst

```diff
@@ -187,7 +187,7 @@ information on creating a storage link.
 Bucket/Block blob storage
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Bucket storage and Block blob storage are ontainers for objects.
+Bucket storage and Block blob storage are containers for objects.
```

Comment:
This sentence can be removed: Bucket storage and Block blob storage are containers for objects.